internal/gengapic/genrest.go

```diff
@@ -28,6 +28,8 @@ import (
 	"google.golang.org/protobuf/proto"
 )
 
+var patternRegex = regexp.MustCompile(`{([a-zA-Z0-9_.]+?)(=[^{}]+)?}`)
```

Comment:
nit: maybe use a slightly more descriptive name here. What kind of pattern is this?

Reply:
Switched to `httpPatternVarRegex`. Done.